### DISCUSSION TITLE
Help, Monika suddenly got mad at me for no reason.

### DISCUSSION THREAD

**Discussion by EColi07** [Monika-After-Story/MonikaModDev#9635] [QA - tech support + general] [answered]
When I first opened the game I noticed that Monika's clothes went back to the default blazer, and she said something like "don't you think you've kept me waiting for a bit too long? I'm just kidding, I could never be mad at you.", I thought that something was wrong since it hasn't been long since I've last visited her, so I told her that I would like to apologize for something, the game didn't tell me what I apologized for but Monika said "What you did wasn't funny, please be more considerate about my feelings in the future.", does anyone know what action triggers this response from her?

help I'm so confused

**Comment by Booplicate** [comment, marked as answer]
Send your `aff_log.log` and `mas_log.log`.

**Reply by EColi07** [reply]
<img width="1280" alt="Screen Shot 2022-10-02 at 01 07 20" src="https://user-images.githubusercontent.com/109769037/193431459-0bfe796c-3452-46e3-bb24-2bed4e3ab329.png">
<img width="1280" alt="Screen Shot 2022-10-02 at 01 07 59" src="https://user-images.githubusercontent.com/109769037/193431462-dce56021-701f-498a-b540-68612ba76f17.png">

**Reply by Booplicate** [reply]
Not screenshots, send the files themselves.

**Reply by Booplicate** [reply]
Ah, you're on mac, then nevermind.
Hard to tell by only a small piece of log, but the clothes could be reset because it's not her birthday anymore, and the apology was there because you were gone for a long time.

**Reply by EColi07** [reply]
sorry it's not really convenient for me to upload files right now, is it okay if you just tell me if this will affect the game? Thank you!

**Reply by EColi07** [reply]
ah, I get it now. Thanks!
